modules/kafka/tests/test_kafka_topic.c

```diff
@@ -78,6 +78,12 @@ _init_topic_names(LogDriver *driver, const gchar *topic, const gchar *fallback_t
   kafka_dd_set_fallback_topic(driver, fallback_topic);
 }
 
+static void
+_set_bootstrap_server(LogDriver *driver, const gchar *bootstrap_server)
```

Comment:
This wrapper does not really provide any value. Simply inlining its content imho better.

Reply:
Ok.